### PR TITLE
fixed indents and errors on creating new weight

### PR DESCRIPTION
**Pull request by base-13**
now it creates a new file if the file doesn't exists
if the file exist and it can't parse it, it asks user to destory the content of file and recreate it

### CODE REVIEW

File: algorithm.py [hunk 1]
```diff
@@ -56,10 +55,36 @@ def genrandweight(size):  # randomly generate weight for first run
...
+        while not choice in ('y', 'n', ''):
+            print(f'invalid input "{choice}"')
+            choice = input('do you want to rewrite file by destorying its content? Y/n [Y]: ').lower()
+        if choice in ('y', ''):
+            print(f'rewriting {filepath}')
+            createfile(filepath, size)
+            print('done')
+            weights = eval(readfile())
+        else:
+            sys.exit()
```
Comment (tusharhero) [inline algorithm.py:86]: What is `sys.exit()` doing?

Comment (Techlord210) [reply algorithm.py:86]: If the user doesn't want to rewrite file the application with exit

Comment (tusharhero) [reply algorithm.py:86]: Alright.

**Review by tusharhero — APPROVED** [review]
Everything else looks good to me!

**Pull request merged**